fix: handle when log is too much (#2)
linuxdeepin/dde-api-proxy: src/dbus-proxy/CMakeLists.txt

set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)

> # NOTE: This project will be deprecate in the feature, but maybe will still needed, now it cannot be compiled with qt6

Reviewer: ```suggestion
# NOTE: While this project is deprecated, it might still be needed for compatibility reasons. Currently it cannot be compiled with Qt6
```

set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)

> find_package(Qt5 COMPONENTS Core DBus REQUIRED)

Reviewer: 这个变动的原因是？原本目的不是同时支持 Qt5 与 Qt6 构建吗？

Author: 里面用的都是Qt5的cmake方法，并没有做兼容

Author: 不过DtkCore也不能qt6啊，还有qdbusxml2cpp-fix也不太确定导出的是不是qt6的
